### PR TITLE
Fix hide API of Transfer & Ingest

### PR DESCRIPTION
**Pull request by Dhwaniartefact**
Connected to https://github.com/archivematica/Issues/issues/1611

Connected to https://github.com/archivematica/Issues/issues/1609

### CODE REVIEW

**Comment by replaceafill** [comment]
@Dhwaniartefact Is this ready for review? The test workflow is failing.

Also I noticed the renamed `hide_request_ingest.curl` file is [pointing to the Transfer status endpoint](https://github.com/artefactual/archivematica-docs/blob/15f06988feb70821cc0cc293f57f5e1a7b0f8066/dev-manual/api/_code/hide_request_ingest.curl#L2). Is that intended?

**Comment by Dhwaniartefact** [comment]
oops forgot to save the changes. Let me push again.

File: dev-manual/api/api-reference-archivematica.rst [hunk 1]
```diff
@@ -224,9 +224,25 @@ Hide
              delete/**
 ===========  =================================   ==============================
 
+Request body parameters:
+
+=================   ================================================================
+
+``transfer_uuid``   UUID OF THE TRANSFER.
+
+=================   ================================================================
```
Comment (replaceafill) [inline dev-manual/api/api-reference-archivematica.rst]: This table doesn't need to be so wide, right? Can we adjust it to the minimum necessary? Same with the table below.

File: dev-manual/api/api-reference-archivematica.rst [hunk 2]
```diff
@@ -224,9 +224,25 @@ Hide
              delete/**
 ===========  =================================   ==============================
 
+Request body parameters:
+
+=================   ================================================================
+
+``transfer_uuid``   UUID OF THE TRANSFER.
```
Comment (replaceafill) [inline dev-manual/api/api-reference-archivematica.rst]:
```suggestion
``transfer_uuid``   UUID of the transfer.
```

File: dev-manual/api/api-reference-archivematica.rst [hunk 3]
```diff
@@ -224,9 +224,25 @@ Hide
...
+==================   ==========================================================
+
+``removed``          Boolean true or false.
+
+==================   ==========================================================
+
 Example request:
 
-.. literalinclude:: _code/hide_request.curl
+.. literalinclude:: _code/hide_request_transfer.curl
```
Comment (replaceafill) [inline dev-manual/api/api-reference-archivematica.rst:241]: :bow:

File: dev-manual/api/api-reference-archivematica.rst [hunk 4]
```diff
@@ -352,9 +368,25 @@ Hide
 ``DELETE``   **/api/ingest/<SIP UUID>/delete/**  *Hides a SIP.*
 ===========  =================================== ==============================
 
+Request body parameters:
+
+=============  ================================================================
```
Comment (replaceafill) [inline dev-manual/api/api-reference-archivematica.rst]: Can we make these tables narrower?

File: dev-manual/api/api-reference-archivematica.rst [hunk 5]
```diff
@@ -352,9 +368,25 @@ Hide
...
+
+``sip_uuid``   UUID OF THE SIP.
```
Comment (replaceafill) [inline dev-manual/api/api-reference-archivematica.rst]:
```suggestion
``sip_uuid``   UUID of the SIP.
```

File: dev-manual/api/_code/hide_request_ingest.curl [hunk 6]
```diff
@@ -0,0 +1,4 @@
+curl -X DELETE \
+    http://my_archivematica_instance.archivematica.org/api/ingest/ \
+    89fd478d-c896-461a-baec-5d1faf661020/delete/ \
+    -H 'Authorization: ApiKey [_your_username_]:[_your_ApiKey_]'
```
Comment (replaceafill) [inline dev-manual/api/_code/hide_request_ingest.curl]: Let's see how the `trailing-whitespace` `pre-commit` hook I'm introducing in https://github.com/artefactual/archivematica-docs/pull/455 modifies these `*.curl` files.

**Review by replaceafill — APPROVED** [review]

File: dev-manual/api/_code/hide_request_ingest.curl (hunk 6 above)
Comment (replaceafill) [reply dev-manual/api/_code/hide_request_ingest.curl]: :facepalm: I meant the [end-of-file-fixer](https://github.com/pre-commit/pre-commit-hooks/blob/main/README.md#end-of-file-fixer) which I introduced later. I just rebased your branch on top of the latest `1.15` and it fixed this correctly.

File: dev-manual/api/api-reference-archivematica.rst (hunk 4 above)
Comment (Dhwaniartefact) [reply dev-manual/api/api-reference-archivematica.rst]: sure

File: dev-manual/api/api-reference-archivematica.rst (hunk 1 above)
Comment (Dhwaniartefact) [reply dev-manual/api/api-reference-archivematica.rst]: sure

File: dev-manual/api/_code/hide_request_ingest.curl (hunk 6 above)
Comment (Dhwaniartefact) [reply dev-manual/api/_code/hide_request_ingest.curl]: Thanks

File: dev-manual/api/api-reference-archivematica.rst [hunk 7]
```diff
@@ -224,15 +224,26 @@ Hide
              delete/**
 ===========  =================================   ==============================
 
+Request body parameters:
+
+=================   ================================================================
+``transfer_uuid``   UUID of the transfer.
+=================   ================================================================
```
Comment (replaceafill) [inline dev-manual/api/api-reference-archivematica.rst]:
```suggestion
=================   =====================
``transfer_uuid``   UUID of the transfer.
=================   =====================
```

File: dev-manual/api/api-reference-archivematica.rst [hunk 8]
```diff
@@ -224,15 +224,26 @@ Hide
...
+
+Response definitions:
+
+==================   ==========================================================
+``removed``          Boolean true or false.
+==================   ==========================================================
```
Comment (replaceafill) [inline dev-manual/api/api-reference-archivematica.rst]:
```suggestion
===========   ======================
``removed``   Boolean true or false.
===========   ======================
```

File: dev-manual/api/api-reference-archivematica.rst [hunk 9]
```diff
@@ -352,15 +363,25 @@ Hide
 ``DELETE``   **/api/ingest/<SIP UUID>/delete/**  *Hides a SIP.*
 ===========  =================================== ==============================
 
+Request body parameters:
+=============  ================================================================
+``sip_uuid``   UUID of the SIP.
+=============  ================================================================
```
Comment (replaceafill) [inline dev-manual/api/api-reference-archivematica.rst]:
```suggestion
=============  ================
``sip_uuid``   UUID of the SIP.
=============  ================
```

File: dev-manual/api/api-reference-archivematica.rst [hunk 10]
```diff
@@ -352,15 +363,25 @@ Hide
...
+
+Response definitions:
+
+==================   ==========================================================
+``removed``          Boolean true or false.
+==================   ==========================================================
```
Comment (replaceafill) [inline dev-manual/api/api-reference-archivematica.rst]:
```suggestion
===========   ======================
``removed``   Boolean true or false.
===========   ======================
```

**Review by replaceafill — APPROVED** [review]

**Pull request merged**